middlewares/stripPrefixRegex_test.go

```diff
@@ -73,18 +73,16 @@ func TestStripPrefixRegex(t *testing.T) {
 				actualPath = r.URL.Path
 				actualHeader = r.Header.Get(ForwardedPrefixHeader)
 			})
-
 			handler := NewStripPrefixRegex(handlerPath, testPrefixRegex)
-			server := httptest.NewServer(handler)
-			defer server.Close()
 
-			resp, err := http.Get(server.URL + test.path)
-			require.NoError(t, err, "%s: unexpected error.", test.path)
+			req := testhelpers.MustNewRequest(http.MethodGet, "http://localhost"+test.path, nil)
+			resp := &httptest.ResponseRecorder{Code: http.StatusOK}
+
+			handler.ServeHTTP(resp, req)
 
-			assert.Equal(t, test.expectedStatusCode, resp.StatusCode, "%s: unexpected status code.", test.path)
+			assert.Equal(t, test.expectedStatusCode, resp.Code, "%s: unexpected status code.", test.path)
```

Comment:
AFAICS, you can still see which tests failed but cannot see the details of individual tests.

I think there are a few alternatives:

- Remove all but the failing tests if you want to see the error details.
- Switch to the terminal for failing tests.
- Switch to a different IDE/editor, either for the failing tests or permanently.

Visual Studio Code (my IDE of choice) is pretty good, though not perfect. There are a few things which cannot be done as nicely as with the Go CLI toolchain; yet, I refrain from letting those shortcomings leak into the code base.

I'll approve the PR since this is my only concern left and I don't want to block it from getting merged any longer (assuming that @emilevauge is fine with it too). Nevertheless, I'm still very much against this change, and even more when it starts to spread throughout the code base over time.

(Side note: Could you please consider filing a bug report on the Gogland issue tacker? IMHO, JetBrains should fix this problem ASAP.)